sos/plugins/manageiq.py

```diff
@@ -73,6 +73,8 @@ def setup(self):
         ])
         self.add_copy_spec("/var/log/tower.log")
 
+        self.add_copy_spec("/etc/manageiq/postgresql.conf.d/*.conf")
```

Comment:
Is there nothing else in `/etc/manageiq/` that's worth collecting?

Reply:
Nope, it looks like it was only added to hold additional postgres confs as of the latest major release.  Thanks for the inquiry though!

Reply:
In future, if you're adding a new item following an existing `add_copy_spec()` call, please just add it as a list - there's no need to make duplicate calls:

```python
                self.add_copy_spec([
                    "/var/log/tower.log",
                    "/etc/manageiq/postgresql.conf.d/*.conf"
                ])
```

It's more readable and makes the code easier to modify in future. I will fix this up as I merge in this case.

Reply:
Thanks for the pointers, I'll try and keep that in mind next time.